geometry_msgs/Pose[] pose
---
geometry_msgs/Point[] gripper_nearest_point_to_obstacle
geometry_msgs/Vector3[] obstacle_surface_normal
float64[] gripper_distance_to_obstacle
